build.gradle.kts

```diff
@@ -187,7 +187,7 @@ val checkGitAttributes by tasks.registering {
                 .map { it.trim() }
                 .filter { it.isNotEmpty() && it.first() !in commentChars }
                 // The patterns are located before the first whitespace.
-                .mapTo(mutableSetOf()) { line -> line.takeWhile { !it.isWhitespace() } }
+                .mapTo(mutableSetOf()) { it.substringBefore(" ") }
```

Comment:
This is a bit different as it does not handle other whitespace characters like `\t`. But I don't know if this is relevant. (@sschuberth)

Reply:
It *is* relevant and breaks the tests, because tabs are used as spacers in that file.